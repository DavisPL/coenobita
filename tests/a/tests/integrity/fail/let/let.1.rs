#[allow(unused_variables)]

fn main() {
    #[cnbt::tag({c,bin}{bin})]
    let x = 5;

    // We shouldn't be able to assign a value with explicit flow
    // set {bin} to a variable with explicit flow set {c,bin}
    #[cnbt::tag({bin}{bin})]
    let y = x;
}