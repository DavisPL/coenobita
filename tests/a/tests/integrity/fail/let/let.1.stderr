error: expected {bin}{bin}, found {bin,c}{bin}
  --> tests/integrity/fail/let/let.1.rs:10:13
   |
10 |     let y = x;
   |             ^
